backend/src/impl/language_code.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from iso639 import languages
+
+
+@dataclass
+class LanguageCode:
+    name: str
+    iso1_code: str
+    iso3_code: str
+
+
+_additional_language_codes: list[LanguageCode] = [
+    LanguageCode("Other", "other", "other"),  # for languages not in ISO396
+    LanguageCode("None", "none", "none"),  # for other modalities
+]
+
+
+def get_language_codes():
+    """getter for language codes data"""
+    language_codes = [
+        LanguageCode(lang.name, lang.part1, lang.part3)
+        for lang in list(languages)
+        if len(lang.part3) > 0
+    ]
+    language_codes.sort(key=lambda x: (x.iso3_code, x.name))
```

Comment:
I think it may be more user-friendly if we sort it by `x.name` because most users are going to choose based on the name. What do you think?

Reply:
Thanks for the explanation! That makes sense.

Reply:
Hi, is this PR ready to merge?